packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -61,6 +61,14 @@ export interface ITooltipProps extends IPopoverSharedProps, IIntentProps {
      * @default 100
      */
     transitionDuration?: number;
+
+    /**
+     * Whether to apply minimal styles to this tooltip, which includes removing
+     * the arrow and adding `Classes.MINIMAL` to minimize and accelerate the
+     * transitions.
+     * @default false
+     */
+    minimal?: boolean;
```

Comment:
this should be moved from `IPopoverProps` in `popover.tsx` to `IPopoverSharedProps` in `popoverSharedProps.ts`. Then you don't have to add it here.

Reply:
Hey @adidahiya - thanks for your feedback! Commit [ab11371fb3fe26d67c63f6e0934a869d023689b0](https://github.com/palantir/blueprint/pull/4374/commits/ab11371fb3fe26d67c63f6e0934a869d023689b0) addresses your suggestions.